backend/src/v2/component/launcher_v2.go

```diff
@@ -673,7 +714,7 @@ func localPathForURI(uri string) (string, error) {
 func prepareOutputFolders(executorInput *pipelinespec.ExecutorInput) error {
 	for name, parameter := range executorInput.GetOutputs().GetParameters() {
 		dir := filepath.Dir(parameter.OutputFile)
-		if err := os.MkdirAll(dir, 0644); err != nil {
+		if err := os.MkdirAll(dir, 0777); err != nil {
```

Comment:
Why changing the permission?

Reply:
I'm not entirely sure why, but if we keep the previous permission, the code doesn't have the permission to write to these created folders.

Reply:
We can check back on it later.